sdk/testutil/server.go

```diff
@@ -342,7 +342,6 @@ func (s *TestServer) Stop() error {
 		return err
 	case <-time.After(10 * time.Second):
 		s.cmd.Process.Signal(syscall.SIGABRT)
-		s.cmd.Wait()
```

Comment:
I think we do still want to wait for the process to exit in this case, because there is a defer at the top of this function, which is responsible for removing the temporary directory. We want to make sure the process has exited before we try and remove it to make sure the cleanup succeeds.

I think we can do that by blocking on a read from the `waitDone` channel instead of calling `s.cmd.Wait()`

```go
<-waitDone
return ...
```

Reply:
Hello @dnephin that totally makes sense. Can you check the last commit ?